### DISCUSSION TITLE
Add multiple lines between two nodes

### DISCUSSION THREAD

**Discussion by d-batmanov** [xyflow/xyflow#1459] [Q&A] [answered]
Hey!
I can't figure out how you can add multiple lines between two nodes.
I'll try to show what I need in the pictures.
Thanks!
<img width="571" alt="Screenshot 2021-08-21 at 18 32 52" src="https://user-images.githubusercontent.com/25131782/130327083-6341c0aa-c8d3-461d-8a18-fbf9823e6641.png">

**Comment by datoslabs** [comment, marked as answer]
You need to create custom nodes.  The [Custom Node example](https://reactflow.dev/examples/custom-node/) shows how to do this.
